go.mod

```
@@ -1,54 +1,84 @@
-// TODO: Replace module name
 module github.com/vshn/exoscale-metrics-collector
 
 go 1.18
 
 require (
-	github.com/appuio/appuio-cloud-reporting v0.5.0
-	github.com/exoscale/egoscale v0.90.0
+	github.com/ccremer/go-command-pipeline v0.20.0
+	github.com/exoscale/egoscale v0.90.1
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/zapr v1.2.3
-	github.com/stretchr/testify v1.8.0
-	github.com/urfave/cli/v2 v2.11.1
+	github.com/urfave/cli/v2 v2.14.1
```

note, there's already newer versions out there.

---

for some reason I already have the latest version

---

There is [v2.16.3](https://github.com/urfave/cli/releases/tag/v2.16.3) out
```suggestion
	github.com/urfave/cli/v2 v2.16.3
```